docs/RESOURCES.md

```diff
@@ -10,6 +10,7 @@
 * [StackOverflow ](http://stackoverflow.com/questions/tagged/c%23) can be used to search for your problem and see if it has been answered already. You can also ask and answer questions.
 * [/r/csharp](https://www.reddit.com/r/csharp) is the C# subreddit.
 * [@dotnet](https://twitter.com/DotNet) is the official .NET Twitter account.
+* [Gitter](https://gitter.im/exercism/xcsharp) is the C# Gitter room.
```

Comment:
also, I forgot to put #208 in my commit message, but I *think* my pull request should be there.    First time I did this it kicked off automated tests, this time it did not so can't tell if I did this correctly, let me know if there is any issue.